packages/aws_common/pubspec.yaml

```
@@ -14,7 +14,7 @@ dependencies:
   http: ^0.13.4
   meta: ^1.7.0
   stream_transform: ^2.0.0
-  uuid: 3.0.6
+  uuid: ^3.0.7
```

Can you update this to just include 3.0.6 and 3.0.7? This will align in with [the dev-preview version](https://github.com/aws-amplify/amplify-flutter/blob/next/packages/aws_common/pubspec.yaml#L25).

```suggestion
  uuid: ">=3.0.6 <=3.0.7"
```